### PR TITLE
ADEN-2714 Add 3d prefooter to home pages

### PR DESCRIPTION
**Pull request by drets**
Add 3d prefooter to home pages

### CODE REVIEW

File: extensions/wikia/AdEngine/js/spec/AdProviderLiftium.spec.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /*global describe, it, modules, expect*/
 describe('AdProviderLiftium', function () {
 	'use strict';
+
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/spec/AdProviderLiftium.spec.js]: Not needed :)

**Comment by fraszczakszymon** [comment]
LGTM :+1:

**Comment by jacekjursza** [comment]
LGTM

**Comment by drets** [comment]
LGTM :+1:

**Pull request merged**